src/app.ts

```diff
@@ -30,8 +30,11 @@ MongoDriverFactory.build()
         name 
       });
       res.json ({id});
-    })
```

Comment:
It looks like the new route you added removed the closing `})` of the post route, and the line that sets the server up to listen for network requests.